Apps/WebClient/src/Server/Models/TourConfiguration.cs

```diff
@@ -0,0 +1,30 @@
+﻿// -------------------------------------------------------------------------
+//  Copyright © 2019 Province of British Columbia
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+// -------------------------------------------------------------------------
+namespace HealthGateway.WebClient.Server.Models
+{
+    using System;
+
+    /// <summary>
+    /// The Tour Configuration.
+    /// </summary>
+    public record TourConfiguration
+    {
+        /// <summary>
+        /// Gets or sets the tour update date time.
+        /// </summary>
+        public DateTime? LatestChangeDateTime { get; set; }
```

Comment:
Is it necessary to have the property defined as DateTime?  Are we gong to have more than one release per day?  Would DateOnly be simpler?

Reply:
As per the discussion the time will allow for a more accurate UX.